### DISCUSSION TITLE
Artic Fuse Tacones error de registro

### DISCUSSION THREAD

**Discussion by amillans** [community/community#136575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

buenas tardes, intento que esto lo lea el creador de Artic Fuse, addons tacones da continuamente error de registro en la skin, desconozco el problema, sin embargo el addon funciona correctamente, pero el mensaje que sale es muy molesto.
![Error Registro](https://github.com/user-attachments/assets/a9aa38d3-99fb-410f-8b01-a43328720bed)
espero una solución, gracias

**Comment by LiteBrite82** [comment]
Hi @amillans ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try submitting an issue in the [skin.arctic.fuse repository](https://github.com/jurialmunkey/skin.arctic.fuse/issues). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
